montapacking-class.php

```diff
@@ -672,7 +672,16 @@ public static function get_shipping_total($data = null)
 
     public static function shipping_options()
     {
+        if (empty($_POST['montapacking']) || !is_array($_POST['montapacking'])) {
```

Comment:
Voor de zekerheid nog een !isset($_POST['montapacking']) in de if voor het geval hij niet bestaat en warnings geeft.